app/browser/reducers/windowsReducer.js

```diff
@@ -43,17 +39,43 @@ function isModal (browserOpts) {
 
 const navbarHeight = () => {
   // TODO there has to be a better way to get this or at least add a test
+  // TODO try creating a window and measuring the difference between window and content area
+  // and updating this number with that value once the first window is created
   return 75
 }
 
+function clearFramesFromWindowState (windowState) {
+  return windowState
+    .set('frames', Immutable.List())
+    .set('tabs', Immutable.List())
+}
+
+/**
+ * Determine the frame(s) to be loaded in a new window
+ * based on user preferences
+ */
+function getFramesForNewWindow () {
+  const startupSetting = getSetting(settings.STARTUP_MODE)
+  const homepageSetting = getSetting(settings.HOMEPAGE)
+  if (startupSetting === 'homePage' && homepageSetting) {
+    return homepageSetting
+      .split('|')
+      .map((homepage) => ({
+        location: homepage
+      }))
+  }
+  return [ { } ]
+}
+
 /**
  * Determine window dimensions (width / height)
  */
 const setWindowDimensions = (browserOpts, defaults, immutableWindowState) => {
   assert(isImmutable(immutableWindowState))
-  if (immutableWindowState.getIn(['windowInfo'])) {
-    browserOpts.width = firstDefinedValue(browserOpts.width, immutableWindowState.getIn(['windowInfo', 'width']))
-    browserOpts.height = firstDefinedValue(browserOpts.height, immutableWindowState.getIn(['windowInfo', 'height']))
+  const windowInfoState = immutableWindowState.get('windowInfo')
+  if (windowInfoState) {
+    browserOpts.width = firstDefinedValue(browserOpts.width, windowInfoState.get('width'))
+    browserOpts.height = firstDefinedValue(browserOpts.height, windowInfoState.get('windowInfo'))
```

Comment:
Here's the bug causing the height issue 🐛  👀 !!

Reply:
oh good spot @bsclifton, I remember deleting that code 🤦‍♂️